### PR TITLE
Add Indonesian translation for Pod page

### PR DESCRIPTION
**Pull request by wahyuoi**
Addresses #15341

### CODE REVIEW

File: content/id/docs/concepts/workloads/pods/pod.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+untuk info lebih lanjut)
+
+Ketika sesuatu dikatakan memiliki umur yang sama dengan Pod, misalnya saja ruang penyimpanan,
+maka itu berarti akan tetap ada selama Pod tersebut masih ada. Jika Pod dihapus dengan
+alasan apapun, sekalipun Pod pengganti yang identik telah dibuat, semua yang berhubungan
+(misalnya ruang penyimpanan) akan dihapus dan dibuat ulang.
+
+{{< figure src="/images/docs/pod.svg" title="Pod diagram" width="50%" >}}
+
+*Sebuah Pod dengan banyak kontainer, yaitu File Puller dan Web Server yang menggunakan
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod.md]:
```suggestion
*Sebuah Pod dengan banyak kontainer, yaitu _File Puller_ dan _Web Server_ yang menggunakan
```

File: content/id/docs/concepts/workloads/pods/pod.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+   kenyamanan untuk pengguna.
+1. Pemisahan ketergantungan perangkat lunak. Setiap kontainer mungkin memiliki 
+   versi, dibuat dan dijalankan ulang secara independen. Kubernetes mungkin mendukung
+   pembaharuan secara langsung terhadap suatu kontainer, suatu saat nanti.
+1. Mudah digunakan. Penguna tidak diharuskan menjalankan manajer prosesnya sendiri,
+   khawatir dengan sinyal dan propagasi _exit-code_, dan lain sebagainya.
+1. Efisiensi. Karena infrastruktur memegang lebih banyak tanggung jawab, kontainer
+   bisa lebih ringan.
+
+_Kenapa tidak mendukung penjadwalan kontainer berdasarkan affinity?_
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod.md]:
```suggestion
Kenapa tidak mendukung penjadwalan kontainer berdasarkan _affinity_?
```

File: content/id/docs/concepts/workloads/pods/pod.md [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+
+Cara itu bisa menyediakan lokasi yang sama, namun tidak memberikan banyak 
+keuntungan dari Pod, misalnya saja berbagi sumber daya, IPC, jaminan berbagi nasib
+dan kemudahan manajemen.
+
+## Ketahanan suatu Pod (atau kekurangan)
+
+Pod tidak dimaksudkan untuk diperlakukan sebagai entitas yang tahan lama. 
+Mereka tidak akan bertahan dengan kegagalan penjadwalan, kegagalan mesin,
+atau pengusiran lain, misalnya karena kurangnya sumber daya atau dalam suatu
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod.md]:
```suggestion
atau _eviction_ (pengusiran), misalnya karena kurangnya sumber daya atau dalam suatu
```

File: content/id/docs/concepts/workloads/pods/pod.md [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+bersama dengan masa tenggang.
+1. Pod ditampilkan sebagai "Terminating" ketika tercantum dalam perintah klien
+1. (bersamaan dengan poin 3) Ketika Kubelet melihat Pod sudah ditandai sebagai 
+"Terminating" karena waktu pada poin 2 sudah diatur, ini memulai proses penghentian Pod
+	1. Jika salah satu kontainer pada Pod memiliki 
+	[preStop _hook_](/docs/concepts/containers/container-lifecycle-hooks/#hook-details), 
+	maka akan dipanggil di dalam kontainer. Jika `preStop` _hook_ masih berjalan
+	setelah masa tenggang habis, langkah 2 akan dipanggil dengan tambahan masa tenggang
+	yang sedikit, 2 detik.
+	1. Semua kontainer akan diberikan sinyak TERM. Sebagai catatan, tidak semua kontainer
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod.md]:
```suggestion
	1. Semua kontainer akan diberikan sinyal TERM. Sebagai catatan, tidak semua kontainer 
```

File: content/id/docs/concepts/workloads/pods/pod.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+
+Secara umum, masing-masing Pod tidak dimaksudkan untuk menjalankan beberapa 
+aplikasi yang sama.
+
+Penjelasan lebih lengkap bisa melihat [The Distributed System ToolKit: Patterns for Composite Containers](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns).
+
+
+## Alternatif pertimbangan
+
+_Kenapa tidak menjalankan banyak program dalam satu kontainer (Docker)?_
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod.md]:
```suggestion
Kenapa tidak menjalankan banyak program dalam satu kontainer (Docker)?
```

File: content/id/docs/concepts/workloads/pods/pod.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+akan dikirim ke setiap proses dan Pod akan dihapus dari API server. Jika Kubelet 
+atau kontainer manajer dijalankan ulang ketika menunggu suatu proses dihentikan,
+penghentian tersebut akan diulang dengan mengembalikan masa tenggang senilai semula.
+
+Contohnya sebagai berikut:
+
+1. Pengguna mengirim perintah untuk menghapus Pod, dengan masa tenggang (30 detik)
+1. Pod dalam API server akan diperbarui dengan waktu dimana Pod dianggap "mati"
+bersama dengan masa tenggang.
+1. Pod ditampilkan sebagai "Terminating" ketika tercantum dalam perintah klien
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod.md]:
```suggestion
1. Pod ditampilkan dalam status "Terminating" ketika tercantum dalam perintah klien
```

File: content/id/docs/concepts/workloads/pods/pod.md [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+yang sama. Dalam mesin, Pod yang dihentikan paksa akan tetap diberikan sedikit masa 
+tenggang sebelum dihentikan paksa.
+
+Penghentian paksa dapat menyebabkan hal berbahaya pada beberapa Pod dan seharusnya
+dilakukan dengan perhatian lebih. Dalam kasus StatefulSet Pods, silakan melihat 
+dokumentasi untuk [penghentian Pod dari StatefulSet](/docs/tasks/run-application/force-delete-stateful-set-pod/).
+
+## Hak istimewa untuk kontainer pada Pod
+
+Setiap kontainer dalam Pod dapat mengaktifkan hak istimewa, dengan menggunakan tanda
```
Comment (irvifa) [inline content/id/docs/concepts/workloads/pods/pod.md]:
```suggestion
Setiap kontainer dalam Pod dapat mengaktifkan hak istimewa (mode _previleged_, dengan menggunakan tanda
```

**Comment by irvifa** [comment]
I forgot to say that after you finished to address the comments and the reviewer add lgtm label I'll need your help of squashing the commits into one and change the commits message to "Translating Pod to ID Localization"
/cc @girikuncoro

**Comment by irvifa** [comment]
@wahyuoi can you please squashing the commits into one commits and change the commit message? By squashing you stil need to force push to your own branch. And for commit messages hope this help https://gist.github.com/matthewhudson/1475276

**Comment by wahyuoi** [comment]
Hi @irvifa,
Already squashed into 1 commit. Thank you :+1:

**Comment by irvifa** [comment]
Thank you, if you’re willing we are more than happy to receive your further contrib by picking another from the list 😄

**Pull request merged**